tools/flowc/backends/common.flow

```diff
@@ -341,7 +341,7 @@ getFcBackendConfigs(config : CompilerConfig) {
 					"Wrong format of JS dependency: '" + dep + "'. Should look like one of the following:\n" +
 					"\tnpm[@braintree/sanitize-url;NativeHost]\n" + 
 					"\tyarn[@braintree/sanitize-url;NativeHost]\n" + 
-					"\tfile[/home/user/TS/lib.ts;NativeHost]", 
+					"\tfile[dir1/dir2/natives_sources.ts;NativeHost]", 
```

Comment:
What is this path?

Reply:
It is an example path, maybe I should rather write 
`file[some_dir1/some_dir2/some_natives_sources.ts;NativeHost]`
to express that there's just a path to some .ts file